From 902215920bbd5f9be1081a5f5f6c7178faa781c9 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 28 Sep 2016 16:19:06 +0300
Subject: [PATCH 0533/1345] dts: mvpp2x: ioremap only GOP part of NetComplex

commit  ef5b4c13603684d17b32deae504d902919f98b5a from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- Only GOP part of NetComplex should be ioremapped by driver.

Change-Id: Ibba4c5a3dc91b3c44d6a86f47a134977abf2ce45
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33186
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi |    2 +-
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index 7ac125d..dc9dd86 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -268,7 +268,7 @@ ppv22@000000 {
 	      <0x130600 0x200>,  /* FCA - flow control regs*/
 	      <0x130e00 0x100>,  /* MSPG - GMAC regs */
 	      <0x130f00 0x100>,  /* MSPG - XLG MAC regs */
-	      <0x441000 0x1000>;  /* RFU-1 Regs */
+	      <0x441100 0x100>;  /* RFU-1 Regs */
 	reg-names = "pp", "xmib", "led", "smi", "tai", "xsmi", "mg", "mspg", "xpcs",
 		    "fca", "gmac", "xlg", "rfu1";
 	clocks = <&cps_syscon0 1 3>, <&cps_syscon0 1 18>, <&cps_syscon0 1 9>,
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index d3d3247..7317fdb 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -354,7 +354,7 @@ ppv22@000000 {
 	      <0x130600 0x200>,  /* FCA - flow control regs*/
 	      <0x130e00 0x100>,  /* MSPG - GMAC regs */
 	      <0x130f00 0x100>,  /* MSPG - XLG MAC regs */
-	      <0x441000 0x1000>;  /* RFU-1 Regs */
+	      <0x441100 0x100>;  /* RFU-1 Regs */
 	reg-names = "pp", "xmib", "led", "smi", "tai", "xsmi", "mg", "mspg", "xpcs",
 		    "fca", "gmac", "xlg", "rfu1";
 	clocks = <&cpm_syscon0 1 3>, <&cpm_syscon0 1 18>, <&cpm_syscon0 1 9>,
-- 
1.7.9.5

